rs/sns/governance/canister/canister.rs

```diff
@@ -726,6 +727,24 @@ fn add_maturity_(request: AddMaturityRequest) -> AddMaturityResponse {
     governance_mut().add_maturity(request)
 }
 
+/// Triggers a refresh of the cached upgrade steps for testing
+#[cfg(feature = "test")]
+#[export_name = "canister_update refresh_cached_upgrade_steps"]
+fn refresh_cached_upgrade_steps() {
```

Comment:
I still think this could be called just `refresh_upgrade_steps`, but leave it up to you to make the call.